### PR TITLE
Bcook dev

### PR DESCRIPTION
**Pull request by gauravj09**

### CODE REVIEW

File: backend_models/cook.php [hunk 1]
```diff
@@ -1,14 +1,29 @@
 <?php
 
-  	require_once(dirname(__DIR__)."/Database/dbAPI.php");
+	require_once(dirname(__DIR__)."/Database/dbAPI.php");
 	require_once(dirname(__DIR__)."/Backend_Models/employee.php");
```
Comment (Ajinkya-Nawarkar) [inline backend_models/cook.php]: Will have to include order.php

File: backend_models/cook.php [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
+
+			function getOrder()
+			{
+				return $this->database->get_order();
+			}
+			
+			function setOrderReady($OID)
+			{
+				$this->order = new Order($OID);
+				$this->order->setOrderReady();
```
Comment (Ajinkya-Nawarkar) [inline backend_models/cook.php]: no such method exists in order class, use setIsReady()

Comment (Ajinkya-Nawarkar) [inline backend_models/cook.php]: No such method exists in Order class. Instead use setIsReady() and pass true in parameters

File: backend_models/cook.php [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
+			private $order;
+			private $database;
+
+			function __construct($EID)
+			{	
+				$this->EID = $EID;
+				$this->database = new dbAPI;
+			}	
+
+			function getOrder()
```
Comment (Ajinkya-Nawarkar) [inline backend_models/cook.php]: get which order? (function name)

Comment (gauravj09) [reply backend_models/cook.php]: So I get the other two errors and I will have them fixed, but I am confused why getOrder() as the function is an error.  I am assuming that the database has a list of all orders and I am calling for the most recent order with the assumption that orders function like a queue. First in first out.

Comment (Ajinkya-Nawarkar) [reply backend_models/cook.php]: Yes you're right. A title like getRecentOrder() tells me that you're looking at the top order in the table.

Comment (gauravj09) [reply backend_models/cook.php]: Okay, I thought there were logical errors in my function. I will have a more descriptive function name.

File: backend_models/cook.php [hunk 4]
```diff
@@ -1,14 +1,29 @@
...
+			{
+				return $this->database->get_order();
```
Comment (Ajinkya-Nawarkar) [inline backend_models/cook.php]: same here

**Pull request merged**